jsonapi.gemspec

```diff
@@ -0,0 +1,16 @@
+version = File.expand_path('../JSONAPI_VERSION', __FILE__).strip
```

Comment:
I've normally seen this as a constant in `lib/gem_name/version.rb` what's the reason for doing it this way?